src/auto/accel_group.rs

```diff
@@ -1,4 +1,4 @@
-// This file was generated by gir (b7f5189) from gir-files (71d73f0)
+// This file was generated by gir (b7f5189) from gir-files (71d73f0+)
```

Comment:
I deleted it and rechecked it out, tried deleting `gir-files/target` and rebuilt everything, nothing I have is really doing the trick here. I've seen this show `-dirty`, but never a `+` so I don't even know what that means.